Portfolio/src/Wrapper.vue

```diff
@@ -0,0 +1,34 @@
+<script setup lang="ts">
+</script>
+
+<template>
+  <div class="styled-wrapper">
+
+    <slot></slot>
+  </div>
+</template>
+
+<style scoped>
+.styled-wrapper {
```

Comment:
I would name this simply `wrapper`. The word styled is completely arbitrary information in my opinion, keep your code as simple as possible whenever possible.

Reply:
Done!